core-services/openshift-monitoring/alertmanager-main_secret_template.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+objects:
+- apiVersion: v1
+  kind: Secret
+  metadata:
+    name: alertmanager-main
+    namespace: ${namespace}
+  stringData:
+    alertmanager.yaml: |-
+      global:
+        resolve_timeout: 5m
+        slack_api_url: "${slack_api_url}"
+      route:
+        group_by: ['alertname', 'job']
+        group_wait: 30s
+        group_interval: 5m
+        repeat_interval: 12h
+        receiver: default
+        routes:
+        - match:
+            severity: critical
+          receiver: slack_general
+        - match:
+            alertname: DeadMansSwitch
```

Comment:
Did this exist in 3.11?

Reply:
Yes. It has been there since I joined DPTP. :)